gavel/templates/admin.html

```diff
@@ -87,7 +184,7 @@ <h2>Judges</h2>
   </div>
   <div class="info">
     <table id="annotators">
-      <thead>
+      <!-- <thead>
```

Comment:
I would recommend you bring back the original admin view page

Reply:
Done!